website/content/docs/auth/kerberos.mdx

```diff
@@ -233,3 +233,64 @@ client.
 The Kerberos auth method has a full HTTP API. Please see the
 [Kerberos auth method API](/vault/api-docs/auth/kerberos) for more
 details.
+
+## Workflows
+
+Refer to the following workflow examples for Kubernetes auth method usage:
```

Comment:
Is this in the wrong file? How does Kubernetes auth relate to kerberos?

Reply:
Thanks for catching. It was an accident change I didn't revert earlier.